src/scrabble.js

```diff
@@ -1,5 +1,49 @@
-function scrabble() {
+
+const letterValues = {
+  'A': 1, 'E':1, 'I':1, 'O':1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+  'D': 2, 'G': 2, 
+  'B': 3, 'C': 3, 'M': 3, 'P': 3,
+  'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+  'K': 5,
+  'J': 8, 'X': 8,
+  'Q': 10, 'Z': 10,
+}
+
+function scrabble(word) {
   // write code here
+  if (word == null){
+    return 0;
+  }
+  let score = 0;
+  let wordMultiplier =1;
+
+  for (let i = 0; i < word.length; i++){
+    const char = word[i];
+
```

Comment:
there are no comments in your code, it's very hard to understand and follow the logic, takes too much effort; suggest adding comments explaining what your code is doing